package.json

```diff
@@ -24,17 +24,18 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bower-less": "^0.0.1",
+    "bower-less": "0.0.1",
     "mkdirp": "^0.5.0",
     "npm-less": "^1.0.0",
     "package-lookup": "^0.1.2",
-    "resolve": "^1.0.0",
+    "resolve": "^1.1.0",
     "resrcify": "^1.0.1",
     "rework": "^1.0.0",
     "rework-assets": "^1.0.0",
-    "rework-bower": "^0.0.1",
+    "rework-bower": "0.0.1",
     "rework-npm": "^1.0.0",
     "rework-vars": "^3.1.1",
+    "sane-scaffold": "^1.0.5",
```

Comment:
i did not do that and i dont know how that happened.
I will revert it. ...strange